xChrome.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.1.0.0'
+ModuleVersion = '1.0.2.0'
```

Comment:
Yup, we shouldn't be doing it here. Version doesn't match what's on PSGallery cause we've updated version to put the build number on the third place and there was no release since that time. The next version of this module published to PSGallery should be 1.1.0.0 (unless there will be some breaking changes implemented till that time - then we would increase to 2.0.0.0)